---
title: "Alpha-taxonomy in world mammals"
output:
  pdf_document:
    latex_engine: xelatex
    fig_caption: true
fontsize: 12pt
geometry: margin = 1in, a4paper
mainfont: Times New Roman
csl: harvard-university-of-cape-town-rvm_2017-08-15.csl
bibliography: Spp_speciation_deliverable.bib
header-includes:
  - \usepackage{lipsum}
  # Institutional logos as a wrapping figure
  - \usepackage{wrapfig}
  # Hanging indentation for references §
  - \usepackage{hanging}
  # Formatting
  - \setlength{\parindent}{30pt}
  - \setlength{\parskip}{0pt}
  - \renewcommand{\baselinestretch}{1.5}
---

```{r TODO, include = FALSE}
# TODO -------------------------------------------------------------------------
# 2017-08-25 07:30

# √ fix in text species name r execs
# - branch and pull for updated .bib
# √ fill in (or remove) "elaborate"s
# √ proof read
```

```{r msw3-script-setup, include = FALSE}
rm(list = ls())
if (!require(pacman)) install.packages("pacman")
library(pacman)
p_load(here)
here()
```

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE,
                      warning = FALSE,
                      message = FALSE)
knitr::opts_knit$set(root.dir = paste(here()))
```

```{r source-msw3-script, include = FALSE, cache = FALSE}
# (Can't cache (magrittr::not() seems to break?))
source("msw3_analyses.R")
# FIXME: use SE versions of `dplyr` verbs; e.g. filter()` => `filter_()`
p_load(tidyverse, magrittr, here, stringr, glue, rjson)
```

```{r region-species}
# Region information -----------------------------------------------------------

# Cape region
cape_msw <- filter(msw,
                   TaxonLevel == "SPECIES",
                   Date >= 1975,
                   str_detect(Distribution, "Cape"))
# Cape "all_column" search
cape_sl_msw <- msw %>%
    search_all_columns("[Ww]estern [Cc]ape") %>%
    filter(TaxonLevel == "SPECIES",
           Date >= 1975,
           Order %in% get_the_coolest(msw, "Order")) %>%
    select(ID, Order, Genus, Species, CitationName, Distribution)

# South Africa
SA_sl_msw <- msw %>%
    search_all_columns("[Ss]outh [Aa]frica") %>%
    filter(TaxonLevel == "SPECIES",
           Date >= 1975) %>%
           #Order %in% get_the_coolest(msw, "Order")) %>%
    select(ID, Order, Genus, Species, CitationName, Distribution)

# Africa
afr_msw <- filter(msw,
                  TaxonLevel == "SPECIES",
                  Date >= 1975,
                  str_detect(Distribution, "Africa"))

# Appearing in Ann. Transvaal Mus. after 1975
transv_msw <- filter(msw,
                     TaxonLevel == "SPECIES",
                     Date >= 1975,
                     str_detect(CitationName, "Transvaal"))

# Search "Afr" in all_columns
afr_sl_msw <- msw %>%
    search_all_columns("[Aa]fr") %>%
    filter(TaxonLevel == "SPECIES",
           Date >= 1975) %>%
           #Order %in% get_the_coolest(msw, "Order")) %>%
    select(ID, Order, Genus, Species, CitationName, Distribution)

```


\begin{wrapfigure}{r}{100pt}  % this figure will be at the right
    \includegraphics[width=1.5cm, height=1.5cm]{uct_logo.png} \hfill
    \includegraphics[width=1.5cm, height=1.5cm]{biosci_logo_crop.png}
\end{wrapfigure}

\par \noindent \textbf{R. van Mazijk} VMZRUA001@myuct.ac.za

\footnotesize
\par \noindent `r format(Sys.time(), "%Y-%m-%d")`
\par \noindent BSc Hons Biological Sciences, Species & speciation module deliverable
\par \noindent For A/Prof A.M. Muasya

\par \noindent \hrulefill

\hfill


\normalsize

\noindent Herein I discuss the progress of mammalian alpha-taxonomy from 1975 to 2005 CE.. *Mammal Species of the World, 3rd Edition* (MSW3) [@MSW3], a database of mammalian taxa and information concerning their treatments from 1702 to 2005 is the major informative dataset for this discussion. First, I focus on the methods employed to diagnose and delimit these mammal species and how these have changes over the the past 30 to 40 years. Second, I discuss the new species and groups discovered in this period. Third, I will attempt to draw conclusions, if possible, about the speciation mechanisms at work in mammal populations based on the evidence use to delimit those mammal species. I will frame these discussions in the context of the species concepts (or lack thereof) typically used in mammalogy and mammalian taxonomy.


## Mammalogical species concepts

The species concept or criteria a taxonomist follows can affect the number of species delimited. Assuming there is reality to species, and that two lineages' being different species is grounded in reality, species concepts enable us to estimate when this occurs. Following the reasoning of @DeQueiroz2011, different species concepts each deal with different lines of empirical evidence. These each describe different aspects of the speciation process. As such, each has criteria for delimiting two different species based on some property of each lineage that would differ between them if they were separate species [@DeQueiroz2011]. Consequently, different species concepts each encapsulate *some* aspect of dissimilarity between two lineages as a consequence of speciation. @DeQueiroz2011 argues that should any one criterion be met, it is parsimonious to assume speciation has begun, and that the bifurcation of the lineages be appropriately regarded with the recognition of two species. This aside, some uniformity in species concepts in taxonomic research is still desirable (if perhaps unattainable [@Padial2006]). The case within mammalian taxonomy is summarised well:

\small

> \textit{Mammalogists, for the most part, have avoided debate on species concepts, but most investigators follow a particular species definition.}

\normalsize

\noindent @Baker2006 make this somewhat alarming statement about the *modus operandi* of mammal taxonomists. It appears that mammal taxonomy, like many fields of biology, has agreed-upon nomenclatural standards, but has little consensus on which species concepts to use when delineating species. This precludes ready comparison of, say, mice species vs bovid species. There is no strict guarantee that the "species" units represent similarly biologically important units at such broad taxonomic scales. Perhaps this is unavoidable, and itself informative. It seems plausible that some biota are significantly functionally distinct from others as to follow different principles of speciation. Indeed, there may be some taxonomic scale (or scales) at which a uniform species concept is impossible, due to the disparity in function of those biota. The Unified Species Concept [@DeQueiroz2011] is thus a useful approximation or solution to this potential problem with our current way of managing taxonomies at higher scales.

Both the non-uniform applicability of species concepts to distantly related taxa *and* idiosyncratic or researcher-specific species concept implementations clearly has impacts on broader taxonomic research, and the management of higher taxonomies themselves. For the most part, mammalogists historically followed (if only loosely) the Biological Species Concept (BSC) [@Mayr]. This is intuitive as mammals are sexually reproducing eukaryotes [@Mayr; @Baker2006]. The Phylogenetic Species Concept (PSC) contrasts with the BSC in focussing on an explicitly evolutionary, lineage based criterion. The PSC thus now dominates in mammalogy, as the most common species concept used or referred to [@Baker2006; @Groves2012]. The PSC solves the problem of the BSC wherein entities widely regarded as ideally being two species would fail to be differentiated reproductively. A common example of this is the ease of hybridisation in polar and grizzly bears which ordinarily occur in allopatry. Under the BSC, they would be one species, but under the PSC, favourably, they are recognised as separate. A similar case is found with what @Baker2006 themselves put forward: the Genetic Species Concept (GSC), wherein a species represents:

\small

> \textit{[...] a group of genetically compatible interbreeding natural populations that is genetically isolated from other such groups.}

\normalsize

\noindent This does not necessarily imply reproductive isolation either, as can be seen in the case of *Thomomys* pocket gophers [@Baker2006]. The transition from majority usage of the BSC to PSC in mammal taxonomy is not so simple, however. There is little explicit consideration of species concepts by the majority of mammal taxonomists [@Baker2006]. And, there is debate. @Groves2012 in particular advocate more strongly for the PSC, even in place of the GSC, in that species defined by the PSC are:

\small

> \textit{[...] 100\% diagnosable: they have fixed heritable differences between them [...] [and the PSC] offer[s] a repeatable, falsifiable definition of species [...]}

\normalsize

\noindent This is evident in the widespread application of the PSC in mammal taxonomy. But nonetheless, the lack of explicit concern for species criteria is troubling. Do mammal taxonomists employ similar methodologies when delimiting species? I discuss this below, despite the nuanced inconsistencies that come about when species concepts differ among mammalogists.


## Lines of evidence

Morphological definitions were the dominant method in mammal species delimitation prior to 1985 [@Baker2006]. Indeed, morphological differences between taxa are still influential in their taxonomy. Furthermore, allopatric populations of mammal that are morphologically distinct (whatever that may entail) are regularly treated as separate species. This concurs with the usage of the PSC above, where non-reproductively isolated taxa (e.g. grizzly and polar bears; lions and tigers) are treated as separate species. Phenetics is still common practice in mammal taxonomy, wherein morphological divergence between populations is numerically described. Species are then diagnosed as clusters in this quantitative trait-space. Phenetics and morphology are often also the starting points of alpha-taxonomic research [e.g. @Lew2006; @Bronner2000], as it is more amenable to efficient, cheap study than molecular systematics.

Early molecular work used in mammal taxonomy employed karyotype studies [e.g. @Bronner2000]---to delineate species on the basis of chromosomal traits [@Baker2006] (e.g. diploid number, chromosome structure, arrangement). Allozymes and DNA sequencing methods arrived later. These molecular methods were costly in their infancy [@Baker2006], but soon became much more practical. The benefits of these methods is their greater objectivity, in that qualitatively scored morphological traits are more subject to researchers' idiosyncrasies (though this applies less so to phenetic methods). The other benefit of characters divorced from observable morphology is that they can allow cryptic species to be uncovered [@Zimmermann2014]. This does, however, contribute to taxonomic inflation. As @Rylands2012 point out (in reference to primates, but this easily pertains to many biota), taxonomic inflation may indeed represent a greater understanding of existing diversity.

Furthermore, molecular systematic methods may reveal morphologically distinct populations to in fact be polymorphs of a single species [@Zimmermann2014]. In fact, @Zimmermann2014 applied a multi-evidence approach to delimiting cryptic mouse lemur species (*Microcebus spp.*) in Madagascar. This integrative approach, using the confluence of ecological, morphological, behavioural, and molecular data, aligns well with the multiple species criteria advocated by @DeQueiroz2011. A commonly used genetic marker in mammal molecular systematics is the mitochondrial sequence *cytochrome-b*. It has been used, e.g., in opossums [@Carvalho2009], and rodent work in applying the GSC [@Baker2006; @Bradley2001]. Multi-locus genetic work is also employed (e.g. in Afro-tropical shrews by @Jacquet2012), increasing the amount of evidence used to delimit species. The use of multiple genes, and indeed even whole genomes [@Murphy2004], in combination with traditional "visible" traits (e.g. ecology, range, morphology) is likely the ideal way to delineate new species [@Zimmermann2014; @DeQueiroz2011].


## Mammal discoveries, 1975---2005

*Mammal Species of the World, 3rd Edition* (MSW3) [@MSW3] is a reputable database of mammalian taxa (species, sub-specific, and super-specific) and information concerning their treatments (e.g. scientific name, authors' name and year described, original publication citation, distribution, etc. [@MSW3]) from 1702 to 2005 CE. This data product is a boon to the discussions here. Using the `.csv` file from the MSW3 webpage (<https://www.departments.bucknell.edu/biology/resources/msw3/>), I explored this dataset in `R` [@RCoreTeam2017], primarily using the `tidyverse` suite of packages for data exploration and visualisation [@Wickham2017]. Not only does MSW3 enable exploration of mammal taxonomic discovery through time, it also facilitated easier literature searches, outlining common journals in which mammal descriptions are published.

Discoveries of new world mammal species over this 30 year period are dominated by rodents and bats (Figure \ref{taxa-thru-time}). It is thus interesting that, in the Cape, only one new mammal species was described in the the period 1975 to 2005 [@Meester1978]: *`r paste(cape_sl_msw$Genus, cape_sl_msw$Species)`*, from the order `r cape_sl_msw$Order`, discovered around Knysna. This shrew species was first described on the basis of its morphology and geographic range, distinct from those of its relatives [@Meester1978].

South African mammal species over this period include *`r paste(cape_sl_msw$Genus, cape_sl_msw$Species)`*, the rodent *`r paste(SA_sl_msw$Genus, SA_sl_msw$Species)[2]`*, and the golden mole *`r paste(SA_sl_msw$Genus, SA_sl_msw$Species)[1]`* (Afrosoricidimorpha) [@Bronner2000]. In the case of the golden mole, karyotypic and phenetic methods were employed, as mentioned above. African mammal fauna discoveries are more diverse: 23 new species in the period 1975---2005, including primates, rodents, shrews, and indeed *`r paste(SA_sl_msw$Genus, SA_sl_msw$Species)[1]`* (Figure \ref{taxa-thru-africa}). It is intuitive that Africa as a whole would have yielded primate discoveries, compared to only South Africa, as there are fewer primate species outside of the tropics. Globally (Figure \ref{taxa-thru-time}), there is a subset of mammal order that are most taxonomically "active", such that there are only eight orders in which $\geq 10$ new species were described, in the period 1975---2005. This is worth bearing in mind when discussing mammalian taxonomy: there is a taxonomic bias to either 1) the rate of discovery in certain taxa, or 2) the species richness of certain taxa. Some combination of 1) and 2) is also plausible.

![\label{taxa-thru-time} \footnotesize Bar-plots of the number of new species (\textit{sensu stricto}, excluding sub-specific additions) described each year in the eight most taxonomically active orders of mammal over the period 1975---2005. Data are from MSW3 (see text) [@MSW3]. Taxonomically "active" here implies that $\geq 10$ new species were described for an order over the 1975---2005 period. These eight orders are those that meet this criterion, such 533 species' discoveries are illustrated here. For reference, Dasyuromorphs are commonly referred to as Tasmanian devils and numbats, Didelphimorphs as opossums, Diprotodonts as koalas and wombats, and Soricomorphs as moles and shrews](ppt_fig.png){ width=65% }

![\label{taxa-thru-africa} \footnotesize Bar-plots of the number of new African mammal species (\textit{sensu stricto}, excluding sub-specific additions) described each year in the three orders of mammal over the period 1975---2005 that had *any* new species found in that period. Not shown is Afrosoricidimorpha, which had one new species: *`r paste(SA_sl_msw$Genus, SA_sl_msw$Species)[1]`*, a golden mole [@Bronner2000] (see text). Data are from MSW3 (see text) [@MSW3].](ppt_fig_africa.png){ width=75% }


## Modes of mammalian speciation

Given the diversity of species concepts used (or ignored) in mammal alpha-taxonomy, and the similarly diverse lines of evidence used to delimit species, it is difficult to draw general conclusions about speciation processes in mammals. Mammals are greatly morphologically and ecologically diverse taxon [@Caro2013]. As such, it can be expected that there is to be a diversity of speciation modes exhibited in mammal populations of different species.

Different ecologically relevant speciation processes (e.g. acoustic differentiation in bats<!--ref!--> and mouse lemurs [@Zimmermann2014]) often manifest in different populations. There is a case-by-case approach needed to understanding speciation processes, as they speciation necessarily happens at small taxonomic scales, as the product of bifurcating lineages. In other words: their is likely a diversity of speciation modes among mammals. This comes as a function of the functional differences between different mammal lineages<!--does this (=>) makes sense here?: (e.g. ordinal differences in Figure \ref{taxa-thru-time})-->. And, different lines of evidence would likely be able to diagnose multiple modes (e.g. sequence data can describe drift and/or reproductive isolation, morphology can describe mating differences and/or habitat preferences). It is worth noting that, as many mammals are vagile vertebrates, allopatric speciation is likely more common than in sessile plants or bacteria. It would be worthwhile exploring the extent to which allopatric speciation has occurred between species delineated on the basis of their allopatry.

It would be insightful, too, to engage in a more thorough literature review---perhaps via aid of text-mining and high performance literature queries---in order to see if there are any trends in the modes of speciation ascertained from different lines of evidence in mammal descriptions.


## Concluding remarks

Molecular methods in mammal species delimitation studies are much more relevant to PSC/GSC based taxonomy. They can indeed disentangle relationships that would go unnoticed under the BSC, or via morphological data alone. Using multiple methodologies together yields more information, and thus more insight, into the nature of mammal species. This has lead, appropriately, to a rise in the number of described species. The influx of high-through-put, next generation sequencing technologies has facilitated genomic and phylogenomic research, which has gained much attention with mammals [@Murphy2004]. Summarily, I have briefly discussed some of the ways mammalian alpha-taxonomy has changed from 1975 to 2005. In doing so, it is clear that the species concepts and methodologies used in this field of taxonomy are not set in stone. It would also be interesting to further explore the connect between speciation mechanisms in mammals and how understandings thereof are affected by data available on a certain taxon.

\noindent \hrulefill

# \small References
\footnotesize \hangparas{15pt}{1}
